### PR TITLE
Reddit feedback - modularize code into separate files

### PR DESCRIPTION
**Pull request by iknowcss**
I will follow up in a bit with an explanation of what I changed.

### CODE REVIEW

**Comment by iknowcss** [comment]
The core of this change is obvious: move the pieces of app1.js into separate files. They are already pretty well separated so the job was easy. You can see them referenced in order in index.html

``` html
<script src="jquery.js"></script>
<script>window.CatApp = {};</script>
<script src="model.js"></script>
<script src="octopus.js"></script>
<script src="adminView.js"></script>
<script src="view.js"></script>
<script src="app1.js"></script>
```

You probably noticed the `window.CatApp = {};` bit. This is part of the solution to the next problem. How can `octopus` reference `model` if it is in a different file? The simplest way to do it is by creating a namespace. That is what `window.CatApp = {};` does. It creates a `CatApp` namespace. Beware though! This is not the _best_ way to solve the problem, but it is the simplest. It is better to understand the concept now before getting into more sophisticated methods.

All the references to variables outside the files are replaced with `window.CatApp.<name of variable>`. For example, in octopus.js you can see I use `window.CatApp.model` instead of `model`.

**Comment by ultrox** [comment]
Hey Cody, thank you so much for taking your time to help me out, I appreciate your effort so much. 
Anyway, I removed 'cursing' from the code, usually I do not curse a lot, occasionally some 'motherfuckers' but sometimes when I have persistent bug, I tend to write some funny comments or/and cursing, will not do that anymore.

Back to the code, I commited your pull request, It's much better now, that file was getting bigger and bigger, and I was thinking to split it up, but had 0 clue how to do that, so it was very low on the list of my priorities.

Basically I'll go thru that tutorial to finish it up and continue improving, I'm starting to think to find a company here around to volunteer so I can be around more expirienced programmer as I think that would be extremely beneficial for my programming, but again JS is not popular here in Bosnia(php is) so next best thing would be to start contributing to some opensource projects.

Anyhow, thanks for your time I learned a lot.

**Comment by iknowcss** [comment]
Glad I could help :) It's really good to see people learning code. I know you will stay consistent!
